versioned_docs/version-8.7/self-managed/identity/deployment/configuration-variables.md

```diff
@@ -51,16 +51,26 @@ import Licensing from '../../../self-managed/react-components/licensing.md'
+For more information on integrating with an OpenID Connect provider, see the [OIDC documentation](/self-managed/setup/guides/connect-to-an-oidc-provider.md).
 
 :::note
+Claims are name/value pairs used to represent an individual identity. Configure your initial claim and value to match the claim used with your OIDC provider. For example, to use your Microsoft Entra unique account ID, set `IDENTITY_INITIAL_CLAIM_NAME` to `oid`, and `IDENTITY_INITIAL_CLAIM_VALUE` to the ID.
+
 Once set, you cannot update your initial claim name and value using environment or Helm values. You must change these values directly in the database.
 :::
 
-| Environment variable           | Description                                                                                                                                     | Default value |
-| ------------------------------ | ----------------------------------------------------------------------------------------------------------------------------------------------- | ------------- |
-| `IDENTITY_INITIAL_CLAIM_NAME`  | The type of claim to use for the initial user. Examples can include `oid`, `name` or `email`.                                                   | `oid`         |
-| `IDENTITY_INITIAL_CLAIM_VALUE` | The value of the claim to use for the initial user. For the default `oid`, the value usually corresponds to the unique ID of your user account. |               |
+| Environment variable                  | Description                                                                                                                                                    | Example value                                     |
+| ------------------------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------- | ------------------------------------------------- |
+| `IDENTITY_INITIAL_CLAIM_NAME`         | The type of claim used to identify the initial user. Examples can include `oid`, `name` or `email`.                                                            | `oid`                                             |
+| `IDENTITY_INITIAL_CLAIM_VALUE`        | The value of the claim specified in `IDENTITY_INITIAL_CLAIM_NAME`. For the default `oid`, the value usually corresponds to the unique ID of your user account. | `f81d4fae-7dec-11d0-a765-00a0c91e6bf6`            |
+| `CAMUNDA_IDENTITY_TYPE`               | Specifies the authentication provider type (`GENERIC` for standard OIDC providers, `MICROSOFT` for Azure AD).                                                  | `GENERIC` or `MICROSOFT`                          |
```

Comment:
Per [MSFT](https://www.microsoft.com/en-us/security/business/microsoft-entra#FAQ), Azure AD is now Microsoft Entra ID. Can we replace there in this docs page?

```suggestion
| `CAMUNDA_IDENTITY_TYPE`               | Specifies the authentication provider type (`GENERIC` for standard OIDC providers, `MICROSOFT` for Microsoft Entra ID).                                                  | `GENERIC` or `MICROSOFT`                          |
```